src/clock.js

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+
+class Clock extends React.Component {
+  constructor(props) {
+    super(props);
+    this.timeZone = props.timeZone;
+    // Initialise component state to contain "date" attribute with current date and time
+    this.state = { date: new Date() };
+    console.log(props.timeZone);
```

Comment:
Let's always remove console logs before pushing our code, as those are just for debugging